lib/solarwinds_apm/config.rb

```diff
@@ -35,70 +35,70 @@ module Config
     # load_config_file
     #
     # There are 3 possible locations for the config file:
-    # Rails default, ENV['APPOPTICS_APM_CONFIG_RUBY'], or the gem's default
+    # Rails default, ENV['SW_APM_APM_CONFIG_RUBY'], or the gem's default
     #
     # Hierarchie:
-    # 1 - Rails default: config/initializers/appoptics_apm.rb
+    # 1 - Rails default: config/initializers/solarwinds_apm.rb
     #     (also loaded  by Rails, but we can't reliably determine if Rails is running)
-    # 2 - ENV['APPOPTICS_APM_CONFIG_RUBY']
-    # 3 - Gem default: <startup_dir>/appoptics_apm_config.rb
+    # 2 - ENV['SW_APM_APM_CONFIG_RUBY']
+    # 3 - Gem default: <startup_dir>/solarwinds_apm_config.rb
     #
     def self.load_config_file
       config_files = []
 
       # Check for the rails config file
-      config_file = File.join(Dir.pwd, 'config/initializers/appoptics_apm.rb')
+      config_file = File.join(Dir.pwd, 'config/initializers/solarwinds_apm.rb')
       config_files << config_file if File.exist?(config_file)
 
       # Check for file set by env variable
-      if ENV.key?('APPOPTICS_APM_CONFIG_RUBY')
-        if File.exist?(ENV['APPOPTICS_APM_CONFIG_RUBY']) && !File.directory?(ENV['APPOPTICS_APM_CONFIG_RUBY'])
-          config_files << ENV['APPOPTICS_APM_CONFIG_RUBY']
-        elsif File.exist?(File.join(ENV['APPOPTICS_APM_CONFIG_RUBY'], 'appoptics_apm_config.rb'))
-          config_files << File.join(ENV['APPOPTICS_APM_CONFIG_RUBY'], 'appoptics_apm_config.rb')
+      if ENV.key?('SW_APM_APM_CONFIG_RUBY')
+        if File.exist?(ENV['SW_APM_APM_CONFIG_RUBY']) && !File.directory?(ENV['SW_APM_APM_CONFIG_RUBY'])
+          config_files << ENV['SW_APM_APM_CONFIG_RUBY']
+        elsif File.exist?(File.join(ENV['SW_APM_APM_CONFIG_RUBY'], 'solarwinds_apm_config.rb'))
+          config_files << File.join(ENV['SW_APM_APM_CONFIG_RUBY'], 'solarwinds_apm_config.rb')
         else
-          AppOpticsAPM.logger.warn "[appoptics_apm/config] Could not find the configuration file set by the APPOPTICS_APM_CONFIG_RUBY environment variable:  #{ENV['APPOPTICS_APM_CONFIG_RUBY']}"
+          SolarWindsAPM.logger.warn "[solarwinds_apm/config] Could not find the configuration file set by the SW_APM_APM_CONFIG_RUBY environment variable:  #{ENV['SW_APM_APM_CONFIG_RUBY']}"
```

Comment:
Minor: It seems a bit redundant: `SW_APM_APM_` (two `APM`s).

Reply:
good find! doesn't need 2 of them